config/phase-0/am-scripts/scripts-config.json

```diff
@@ -203,6 +203,21 @@
                 "lastModifiedDate":1565386615495
              },
              "filename": "template-oidc-claims-script.groovy"
+        },
+        {
+            "payload": {
+                "_id": "c3f22327-1e32-4f18-957f-731aa9c97dd5",
```

Comment:
This is the ID of the scripted node but should be ID of the script: `b480d9f7-5908-45cf-91d1-bc1fe56fe8de`

Reply:
Fixed.